### PR TITLE
AIM OS Abstraction Updates

### PR DESCRIPTION
**Pull request by jnealtowns**
Reviewer: @rlane 

- The functionality from the standalone bigcode OS abstraction module is now a builtin part of AIM.
- Daemon support is now enabled by default.

### CODE REVIEW

File: modules/AIM/module/inc/AIM/aim_time.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+typedef uint64_t aim_time_t;
+
+/**
+ * @brief Monotonic timer value.
+ */
+
+uint64_t aim_time_monotonic(void);
+
+/**
+ * @brief Current time in ms.
```
Comment (rlane) [inline modules/AIM/module/inc/AIM/aim_time.h]: ms or us?

**Comment by rlane** [comment]
Looks fine, only one comment.

**Comment by jnealtowns** [comment]
SELF-ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vbs22.eng.bigswitch.com/abat/2016.02.29.1933-m.infra.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vbs22.eng.bigswitch.com/abat/2016.02.29.1933-m.infra.master/abat.log
/cc @rlane @jnealtowns

**Pull request merged**